mysql-test/suite/innodb/t/innodb_corrupt_bit.test

```diff
@@ -8,6 +8,7 @@
 
 --disable_query_log
 call mtr.add_suppression("Flagged corruption of.* in table .* in .*");
+call mtr.add_suppression("Index ./test/corrupt_bit_test_@1s is corrupted");
```

Comment:
Could we convert the filename-safe table name to the user-specified name `test.corrupt_bit_test_ā` with something like this?
```c++
static void dict_table_open_failed(const table_name_t &name)
{
  my_printf_error(ER_TABLE_CORRUPT,
                  "Table %`.*s.%`s is corrupted."
                  " Please drop the table and recreate.",
                  MYF(ME_ERROR_LOG),
                  int(name.dblen()), name.m_name, name.basename());
}
```
Or would it require a different format variant to be implemented?

Reply:
Please do not use &table_name_t ; Please use pointers instead.
If possible, use C syntax cast instead of C++ syntax cast.

Reply:
Note that when using a table name for error messages inside InnoDB, you have to use conversion functions to change it from filename format to table name format.
You have to use innobase_convert_to_system_charset() when converting filenames to table names.